### PR TITLE
'extension of types'

### PR DESCRIPTION
**Pull request by cquigley2**

### CODE REVIEW

**Comment by tasodorff** [comment]
@cquigley2  you are using an outdated version of the ontology, and are trying to commit versions of files which have since been updated (such as `units.yaml`). please sync your ontology with the master and then resubmit with only your changes.

**Review by tasodorff — APPROVED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -2558,6 +2570,7 @@ ECON:
   - outside_air_relative_humidity_sensor
   - outside_air_damper_percentage_sensor
   - return_air_damper_percentage_command
+  - outside_air_damper_percentage_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:2573]: @cquigley2 please remove -- looks like it is defined on line 2571 already.

Comment (cquigley2) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: removed 2573

**Comment by cstirdivant** [comment]
@cquigley2 please remove the ID line in your ABSTRACT file.

**Pull request merged**